### PR TITLE
CONTRIB-000: Update the FE build tools.

### PR DESCRIPTION
**Pull request by AlviMurtaza**
- Update the FE build tools for the Contrib tracker.
- Update the dependencies for the latest versions with added overrides and dependencies required for the latest devDependencies.
- Verify the updated changes are working with the latest node versions and that the theme is compiled correctly without errors.

### CODE REVIEW

File: web/themes/custom/contribtracker/package.json [hunk 1]
```diff
@@ -45,7 +45,13 @@
     "typescript": "^4.8.4"
   },
   "dependencies": {
-    "select2": "^4.1.0-rc.0"
+    "rimraf": "^5.0.5",
+    "select2": "^4.1.0-rc.0",
+    "uuid": "^9.0.1"
```
Comment (hussainweb) [inline web/themes/custom/contribtracker/package.json:50]: @AlviMurtaza, why are these dependencies needed? Or did you mean to add them to `devDependencies`?

Comment (AlviMurtaza) [reply web/themes/custom/contribtracker/package.json:50]: We were getting a similar error after the updates. https://github.com/isaacs/rimraf/issues/264 Adding this dependency resolved the issue.

Comment (hussainweb) [reply web/themes/custom/contribtracker/package.json:50]: Sure, but my question is if this should be a part of `devDependencies`?

Comment (AlviMurtaza) [reply web/themes/custom/contribtracker/package.json]: Updated

**Review by hussainweb — APPROVED** [review]
Awesome. Thanks for the changes.

**Pull request merged**